offix-web/views/layout.jade

```diff
@@ -2,6 +2,14 @@ doctype html
 html
   head
     title= title
-    link(rel='stylesheet', href='/stylesheets/style.css')
+    link(rel="stylesheet",
+     href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css",
+     integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u",
+      crossorigin="anonymous")
+    link(href="https://fonts.googleapis.com/css?family=Montserrat|Open+Sans", rel="stylesheet")
+    link(rel='stylesheet', href='/stylesheets/styles.css')
+    link(rel="icon", type="image/png", href="img/favicon.png")
```

Comment:
The Copied the favicon from the techx website, and it used a .png, so I just assumed that it was ok to have .png as a favicon. I've converted it to a .ico though.